Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetStreetcodeByFilterHandlerTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Linq.Expressions;
+using FluentAssertions;
+using Moq;
+using Streetcode.BLL.DTO.AdditionalContent.Filter;
+using Streetcode.BLL.MediatR.Streetcode.Streetcode.GetByFilter;
+using Streetcode.DAL.Entities.Media.Images;
+using Streetcode.DAL.Entities.Streetcode;
+using Streetcode.DAL.Entities.Streetcode.TextContent;
+using Streetcode.DAL.Entities.Timeline;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatRTests.StreetCode.Streetcode;
+
+public class GetStreetcodeByFilterHandlerTests
+{
+    private readonly Mock<IRepositoryWrapper> _repositoryWrapper;
```

Comment:
Here will be warning `SA1309 A field name in C# begins with an underscore.`
We left this rule because with use `this` in test projects

Reply:
Do you mean we should just call this property `repositoryWrapper`?

Reply:
yeah and another one too